### DISCUSSION TITLE
Anywhere to find uploaded lists?

### DISCUSSION THREAD

**Discussion by endlesssmoko** [SelfControlApp/selfcontrol#883] [General]
"...or even post it on a website."

Did this ever happen?

I'd like to block as many porn sites as possible in one go and then forget about having to add anymore. Theres mega lists already out there but it doesnt seem possible to edit the blocklist files that are able to be exported?

Thanks!

**Comment by cstigler** [comment]
Re editing a blocklist file: you can import (replaces your current list), modify the list, and re-save!

But yes I'm not aware of any repository of SC blocklists online, so getting a mega-list in another format into SC formatting might be a pain / require technical know-how.
